### PR TITLE
feat(ai-summary): discussion summary in meeting summary

### PR DESCRIPTION
**Pull request by nickoferrall**
Fix https://github.com/ParabolInc/parabol/issues/7488

Demo: https://www.loom.com/share/9dabdb277aca410bacf66fa85dcd5666

### To test

- [ ] Make sure you've added both the `OPEN_AI_API_KEY` and `OPEN_AI_ORG_ID` env vars 
- [ ] Add the `aiSummary` feature flag to the user
- [ ] Create a retro, add a bunch of comments and tasks in the discussion threads
- [ ] When you end the meeting, you should see a `Discussion Summary` in the meeting summary

### CODE REVIEW

File: packages/server/graphql/mutations/endRetrospective.ts [hunk 1]
```diff
@@ -93,6 +94,7 @@ export default {
       return standardError(new Error('Cannot find facilitator stage'), {userId: viewerId})
     }
     const {stage} = currentStageRes
+    await handleCompletedStage(stage, meeting, dataLoader)
```
Comment (nickoferrall) [inline packages/server/graphql/mutations/endRetrospective.ts:97]: When we end the meeting, `handleCompletedStage` isn't called from `navigateMeeting` so the `discussionSummary` isn't generated. To fix, I've added this here in `endRetrospective`

**Comment by nickoferrall** [comment]
> Looks good! I wonder should we show some kind of spinner or loading indicator if summary is not ready yet?

Yeah, I think you're right! I've created an issue as I don't think it'll be straightforward: https://github.com/ParabolInc/parabol/issues/7537

**Review by Dschoordsch — APPROVED** [review]
I think it's good for now. I would like to update the summary once the API requests finished. Maybe we should add these to the meeting subscription and just send them once ready?

I haven't checked the email, but that would also miss these fields, right?

**Comment by nickoferrall** [comment]
>  I would like to update the summary once the API requests finished. Maybe we should add these to the meeting subscription and just send them once ready?

Agreed! This issue should resolve that: https://github.com/ParabolInc/parabol/issues/7537

> I haven't checked the email, but that would also miss these fields, right?

In a later PR (https://github.com/ParabolInc/parabol/pull/7543), I only send the email once the whole summary has been generated which means the discussion summary should also have been generated by then

**Pull request merged**